src/viam/services/vision/vision.py

```diff
@@ -75,6 +99,18 @@ async def get_classifications_from_camera(
     ) -> List[Classification]:
         """Get a list of classifications in the next image given a camera and a classifier
 
+        ::
+
+            camera_name = "cam1"
+
+            # Grab the classifier you configured on your machine
+            my_classifier = VisionClient.from_robot(robot, "my_classifier")
+
+            # Get the 2 classifications with the highest confidence scores from the next
+            # image from the camera
```

Comment:
I think you can merge these lines
```suggestion
            # Get the 2 classifications with the highest confidence scores from the next image from the camera
```